server/protocols/jmap-rfc-8621/src/main/scala/org/apache/james/jmap/core/Capability.scala

```diff
@@ -168,6 +168,7 @@ final case class SubmissionCapability(identifier: CapabilityIdentifier = EMAIL_S
 
 case object SubmissionCapabilityFactory {
   val maximumDelays = Duration.ofDays(1)
+  val MAX_FUTURE_RELEASE_INTERVAL = 86400
```

Comment:
```suggestion
```

Reply:
I fix it already, because I misunderstood what you meant before